pkg/project/registry/project/proxy/proxy_test.go

```diff
@@ -139,3 +141,24 @@ func TestDeleteProject(t *testing.T) {
 		t.Errorf("Expected call to delete-namespace")
 	}
 }
+
+// TestUpdateProject makes sure that the project to namespace conversion does not lose any data
+func TestUpdateProject(t *testing.T) {
+	f := fuzz.New().NilChance(0)
+	p := &api.Project{}
+	fake := &testclient.Fake{}
+	c := fake.Namespaces()
+
+	for i := 0; i < 1000; i++ {
+		f.Fuzz(p)
```

Comment:
I think we also want to start with a namespace, then round trip to project and back. That would catch any new fields in namespace we weren't round tripping